engine/src/main/scala/cromwell/engine/workflow/workflowstore/SqlWorkflowStore.scala

```diff
@@ -56,7 +56,7 @@ case class SqlWorkflowStore(sqlDatabase: WorkflowStoreSqlDatabase) extends Workf
       workflowStoreEntry.workflowDefinition.toRawString,
       workflowStoreEntry.workflowInputs.toRawString,
       workflowStoreEntry.workflowOptions.toRawString,
-      workflowStoreEntry.customLabels.toRawString,
+      workflowStoreEntry.customLabels.toString,
```

Comment:
FYI- Hurray! Centaur caught this bad call `Object.toString()`. One example of _many_:

```java
2017-03-17 17:13:01,231 cromwell-system-akka.dispatchers.engine-dispatcher-41 ERROR - WorkflowManagerActor Workflow 35d07911-6931-4d3a-91be-5007d7b83bbd failed (during MaterializingWorkflowDescriptorState): Workflow input processing failed:
Workflow contains invalid labels JSON: Unexpected character 'o' at input index 0 (line 1, position 1), expected JSON Value:
org.hsqldb.jdbc.JDBCClobClient@28f51798
^
```